README.md

```
@@ -244,6 +245,74 @@ gctools random-posts <apiURL> <adminAPIKey> --count 50 --contentUnit paragraphs
+# Add a dummy author (requires a staff access token) as primary author on ~40% of posts
+gctools seed-demo <apiURL> <staffAccessToken> --addAuthor --authorShare 40
+
+# Seed content and also set navigation automatically (pass an owner/admin staff access token as the key)
+gctools seed-demo <apiURL> <staffAccessToken>
+
+# Preview what would be created without writing anything
+gctools seed-demo <apiURL> <adminAPIKey> --dryRun
+```
+
+> **Navigation & staff access tokens:** Updating the navigation menu writes to Ghost's settings, which integration API keys are not permitted to do. Supply an owner/admin [staff access token](https://docs.ghost.org/admin-api#staff-access-token-authentication) (found on a staff user's profile page, in the same `{id}:{secret}` format) as the `<adminAPIKey>` argument and navigation is written automatically. With an integration key everything else still runs; the navigation step is skipped and the menu JSON is printed for you to paste into Ghost Admin → Settings → Navigation.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)